game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt

```diff
@@ -41,7 +41,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "damage_base"                                     "40 60 80 120 240 480" //OAA
+        "damage_base"                                     "0 40 80 120 240 480" //OAA
```

Comment:
ADAPTIVE STRIKE IS NOT GOOD

Reply:
AGILITY THAT IS

Reply:
this has got to be one of the oddest nerfs I've seen.

Reply:
I played a round of morph and he just did too much damage, Darkonius and I agreed this was the best way of dealing with how powerful adaptive strike is levels 1 and 2